components/mpas-seaice/src/shared/mpas_seaice_column.F

```diff
@@ -1875,7 +1881,10 @@ subroutine column_vertical_thermodynamics(domain, clock)
                dayOfYear, &
                abortFlag, &
                abortMessage, &
-               config_use_prescribed_ice)
+               config_use_prescribed_ice, &
+               config_use_congelation_basal_melt, &
+               config_use_lateral_melt, &
+               config_use_latent_processes)
```

Comment:
What is the computational burden of adding 'if' conditionals for all of these individual processes?  They have to be parsed for each category, on each grid cell with ice, perhaps multiple times per timestep.

Reply:
@eclare108213: Good point. I have recoded this so that there are no conditionals now, just multiplication by a real flag taking values (0,1). I will test performance of these changes as well